db/migrate/20150130201130_add_event_type_id_to_position.rb

```diff
@@ -0,0 +1,5 @@
+class AddEventTypeIdToPosition < ActiveRecord::Migration
+  def change
+    add_column :positions, :event_type_id, :integer
```

Comment:
this looks like your trying to setup an association without using associations? Event Type __has a__ position?